assets/css/less/admin/modal.less

```diff
@@ -0,0 +1,70 @@
+.wpbdp-admin-dialog {
```

Comment:
This modal needs another look to make it look like the design. I'm fine with not adding the second button, but the style needs to look a lot closer (lines go to the edges, spacing...)

Reply:
Will work on those. The modal is currently rendered using the existing forms in he category and tag pages. Will improve the styling :)

Reply:
Modal styling has been updated